native/src/main/java/io/ballerina/stdlib/grpc/protobuf/cmd/GrpcCmd.java

```diff
@@ -233,8 +261,19 @@ private void generateBalFile(String protoPath) {
             }
             LOG.debug("Successfully generated root descriptor.");
             try {
-                dependant = DescriptorsGenerator.generateDependentDescriptor(this.protocExePath,
-                        getAbsolutePath(protoPath), descFilePath.toAbsolutePath().toString());
+                if (importPath.equals("")) {
```

Comment:
How about using `importPath.isBlank()` here?

Reply:
Fixed in 3f03c2b